default/stringtables/en.txt

```diff
@@ -9902,6 +9902,32 @@ Krill Spawner
 SP_KRILL_SPAWNER_DESC
 Spawns [[predefinedshipdesign SM_KRILL_1]] in systems with unowned asteroids which do not already contain a krill monster.  When carried on an unarmed ship, the [[SP_KRILL_SPAWNER]] also provides substantial [[encyclopedia STEALTH_TITLE]] (but will not stack with any other [[encyclopedia STEALTH_TITLE]] - causing  parts).
 
+SP_SOLAR_CONCENTRATOR
+Solar Concentrator
+
+SP_SOLAR_CONCENTRATOR_DESC
+'''The Solar Concentrator uses the phototropic effect of [[SH_ORGANIC]]s to increase the strength of [[SR_WEAPON_2_1]]s. The effectness depends on the brightness of the star at the current location. It is expected to increase the weapons strength between 1.5 and 4.5. The research of [[SHP_SOLARCONNECTION]] suggest to increase the collecting surface to maximize the effect.'''
+
+SHP_SOLARCONNECTION
+Solarweb
+
+SHP_SOLARCONNECTION_DESC
+'''This upgrade for the [[SP_SOLAR_CONCENTRATOR]] is applied by the ships crew. It enables the ship to establish one energy web between ships of the same empire at the same star system. This [[SHP_SOLARCONNECTION]] provides an additional surface to collect photons which are evenly distributed among [[SH_ORGANIC]]s. Theoretical gigantic webs can increase the intensity of [[SR_WEAPON_2_1]]s by 130% under ideal conditions. A more realistic approach suggests 100 concentrators to achieve an increase by 80% while 20 concentrators provide an amplification of 50%. The finely woven [[SHP_SOLARCONNECTION]] collapse when the ship enters a starlane whereas the effects disappears shortly after.''' 
```

Comment:
No need to use triple quotes if you you have single line and it is not starts/ends with spaces or tabs.  Same for other entries.